src/Plugin/migrate/source/Form/UrlImporterForm.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\feeds_migrate\Plugin\migrate\source\Form;
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * The importer form for the url migrate source plugin.
+ *
+ * @MigrateForm(
+ *   id = "url_importer_form",
+ *   title = @Translation("Url Source Plugin Importer Form"),
+ *   form_type = "importer",
+ *   parent_id = "url",
+ *   parent_type = "source",
+ * )
+ */
+class UrlImporterForm extends UrlFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+    $form = parent::buildConfigurationForm($form, $form_state);
+
+    $plugins = $this->getPlugins();
+    foreach ($plugins as $plugin_type => $plugin_id) {
+      // Disable the plugin selection field.
+      $form[$plugin_type . '_wrapper']['id']['#attributes']['disabled'] = TRUE;
+    }
+
+    return $form;
+  }
+
+}
```

Comment:
At first glance, it looks like this form still receives a lot of the form fields not needed at the importer level. I think instead of disabling, I rather not see a selector for the fetcher/parser at all. Is this meant as a temporary solution?

Reply:
Leaving it in as a temporary solution until we've covered more use cases and can definitively say we won't need this!